homeassistant/components/volvooncall/__init__.py

```diff
@@ -112,6 +113,19 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up the Volvo On Call component from a ConfigEntry."""
+
+    if CONF_UNIT_SYSTEM not in entry.data.keys():
+        new_conf = {**entry.data}
+
+        scandinavian_miles: bool = entry.data[CONF_SCANDINAVIAN_MILES]
+        new_conf.pop(CONF_SCANDINAVIAN_MILES)
```

Comment:
Don't remove this item. Just add the missing item. That way the user can roll back and the config will still work for them.

Reply:
```suggestion
```